st2common/st2common/ssh/parallel_ssh.py

```diff
@@ -0,0 +1,212 @@
+# Licensed to the StackStorm, Inc ('StackStorm') under one or more
```

Comment:
Almost seems like this should live in `st2actions`, but I can be convinced otherwise